src/Templates/parts/table-content.twig

```diff
@@ -18,7 +18,7 @@
                 <a href="{{column.url}}" title="{{column.title}}"><span class="glyphicon glyphicon-{{column.value}}">&nbsp;</span></a>
         {% endif %}
       {% else %}
-                {{column | raw}}
+                {{column}}
```

Comment:
I thought it would break icons or links, but it does not. The only thing I
could see is show descriptions, but I don't think that matters too much.
And even then raw is not a good filter, there should be a formatting only
safe filter, which is how the new front-end site works.
On 26 Oct 2015 10:35, "Lloyd Wallis" <notifications@github.com> wrote:

> In src/Templates/parts/table-content.twig
> <https://github.com/UniversityRadioYork/MyRadio/pull/516#discussion_r42977711>
> :
>
> > @@ -18,7 +18,7 @@
> >                  <a href="{{column.url}}" title="{{column.title}}"><span class="glyphicon glyphicon-{{column.value}}">&nbsp;</span></a>
> >          {% endif %}
> >        {% else %}
> > -                {{column | raw}}
> > +                {{column}}
>
> I remember that this will break something, but I can't remember what.
> Helpful, I know.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UniversityRadioYork/MyRadio/pull/516/files#r42977711>.
>